features/step_definitions/mocked_configuration.rb

```diff
@@ -303,19 +292,19 @@
 			]
 		}
 	])
-	page.driver.headers = {'X-op5-mock' => @mock.file}
 end
 
 Before do |scenario|
+  ENV.delete("ci_tmp_mock_file")
   @mock = Mock::Mock.new
 end
 
 After do |scenario|
   case scenario
-  when Cucumber::Ast::Scenario
-    name = scenario.name
-  when Cucumber::Ast::OutlineTable::ExampleRow
+  when Cucumber::Core::Ast::ScenarioOutline
```

Comment:
As of cucumber v3+ we only need to reference scenario.name.

Reply:
True. I don't see that we need this at all then. I'll try to do an overhaul.